### PR TITLE
Add inspectds

### PR DESCRIPTION
**Pull request by pmav99**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by pmav99** [comment]
@conda-forge/help-python, ready for review!

File: recipes/inspectds/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/inspectds-{{ version }}.tar.gz
+  sha256: 8e7cdb3bd52fa98793497507da23bd2b91d41e2f2baa5f2561be10a709d19048
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win or py<=38]
```
Comment (dopplershift) [inline recipes/inspectds/meta.yaml:14]: This looks like pure Python...is there any reason you need to skip Windows?

Comment (pmav99) [reply recipes/inspectds/meta.yaml:14]: The build was failing and I don't know how to make it pass. E.g. 
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=696689&view=logs&jobId=240f1fee-52bc-5498-a14a-8361bde76ba0&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=2fb5b0a7-737c-5f5c-8f3f-f6db6174bacf

**Comment by pmav99** [comment]
@conda-forge/help-python, ready for review!

**Comment by pmav99** [comment]
@conda-forge/help-python, ready for review!

File: recipes/inspectds/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - python >=3.9
+    - poetry-core >=1.0.0
+    - poetry-dynamic-versioning
+    - pip
+  run:
+    - python >=3.9
+    - xarray *
+    - zarr >=2.8
+    - typer *
+    - netcdf4 *
```
Comment (ocefpaf) [inline recipes/inspectds/meta.yaml:30]:
```suggestion
    - xarray
    - zarr >=2.8
    - typer
    - netcdf4
```

**Pull request merged**